### PR TITLE
[#159889136] Add a wrapper to GetServicesByRecipientOKResponse and GetVisibleServicesOKResponse to fix the case of pageSize

### PR DESCRIPTION
**Pull request by lussoluca**
Another victim of Autorest :-(

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
## Affected stories

  * 🐞 [#159889136](https://www.pivotaltracker.com/story/show/159889136): getServicesByRecipient e getVisibleServices non funzionano perché autorest resituisce pageSize ma nel tipo è page_size

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: api_proxy.yaml [hunk 1]
```diff
@@ -232,6 +206,32 @@ paths:
...
+      responses:
+        "200":
+          description: Found.
+          schema:
+            $ref: "#/definitions/ServiceList"
+          examples:
+            application/json:
+              page_size: 1
+              items:
+                - service_id: 2b3e728c1a5d1efa035c
```
Comment (gunzip) [inline api_proxy.yaml]: I guess the example here is wrong as it  should contain the service version as well

Comment (lussoluca) [reply api_proxy.yaml]: at the moment the example is correct. Will be fixed here: https://www.pivotaltracker.com/story/show/159464415

**Comment by cloudify** [comment]
Obsolete, see https://github.com/teamdigitale/italia-backend/pull/292

**Pull request closed without merging**